secretariat/tests/test_models.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
+
+from secretariat.models import Organisation, User
+from secretariat.tests.factories import (
+    MembershipFactory,
+    OrganisationFactory,
+    UserFactory,
+)
+
+
+class TestMembership(TestCase):
+    def test_user_organisations(self):
+        user = UserFactory()
+        for _ in range(5):
```

Comment:
oh woaw, qu'est-ce donc que cette étrangeuh syntaxe ?

Reply:
Je voulais faire 5 fois la même chose, ici créer une organisation relative à cet utilisateur.
Comment faire ? Soit je copie 5 fois la ligne, soit je fais une boucle.
Pour faire une boucle de x éléments tu peux utiliser la fonction range(x) : c'est une fonction qui itère sur les entiers de zéro à x-1.
Et on garde la convention du nom de variable `_` pour les variables intentionnellement non utilisées.
```python
for i in range(6):
   print(i) # affiche 0, 1, 2, 3, 4 et 5

for _ in range(6):
   print('na', end="") # affiche nananananana
```

Reply:
Oui c'est le _ qui m'a perturbée. for i in range(6) c'est la base mais le `_` pour les variables non utilisées, trop fort, ça part direct en #1jour1leçon